agent/src/agent/source/validator.py

```diff
@@ -210,6 +210,26 @@ def validate_connection(self):
             )
 
 
+class SolarWindsValidator(Validator):
+    VALIDATION_SCHEMA_FILE = 'solarwinds.json'
+
+    def validate_connection(self):
+        url = self.source.config['url'] + '/SolarWinds/InformationService/v3/Json/Query?query=SELECT TOP+1+1+as+test+FROM+Orion.Accounts'
+        session = requests.Session()
+        session.auth = (
+            self.source.config[source.APISource.USERNAME],
+            self.source.config[source.APISource.PASSWORD]
+        )
+        try:
+            res = session.get(url, verify=False)
```

Comment:
add timeout to the request otherwise it will just hang infinitely if host is not reachable